asapdiscovery-data/asapdiscovery/data/selectors/pairwise_selector.py

```diff
@@ -16,7 +16,7 @@ class PairwiseSelector(SelectorBase):
 
     def _select(
         self, ligands: list[Ligand], complexes: list[Union[Complex, PreppedComplex]]
-    ) -> list[Union[CompoundStructurePair, DockingInputPair]]:
+    ) -> list[CompoundStructurePair]:
```

Comment:
Since DockingInputPair subclasses CompoundStructurePair now it isn't necessary, right? Also I'm trying to move towards not having the data module depend on the docking module

Reply:
Ok yep makes a lot of sense.